data/map.txt

```diff
@@ -29235,6 +29280,15 @@ planet Ochrescoop
 	bribe 0.1
 	security 0.5
 
+planet "Ogmur's Siphon"
+	attributes arach "coalition station" research
+	landscape land/station36
+	description `This refinery was built shortly after the nearby dry dock station was finished, to ensure the system was self-sufficient to provide the hyperspace fuel needed for the visiting ships. A small fleet of tanker ships is docked on specially marked hangar bays, always ready to bring the fuel elsewhere if needed.`
+	description `	"Ogmur" is the name of the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`
```

Comment:
```suggestion
	description `	"Ogmur" is the name of the gas giant below, named after a giant snake-like creature of Arach mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`
```